pkg/messaging/helper.go

```diff
@@ -0,0 +1,38 @@
+package messaging
+
+import (
+	"fmt"
+	"strconv"
+	"strings"
+
+	"github.com/pkg/errors"
+)
+
+const (
+	PubSubTopic      = "alerts"
+	BotRequestsTopic = "bot_requests"
+
+	numberOfStringsAfterSlash = 2
+)
+
+func ParseHostAndPortFromURL(natsPubSubURL string) (string, int, error) {
```

Comment:
Maybe use `url.Parse` with `net.SplitHostPort` instead of this? ([link](https://stackoverflow.com/questions/62083272/parsing-url-with-port-and-without-scheme))